public/js/menu.js

```diff
@@ -1,66 +1,67 @@
-window.onload = () => {
-    document.getElementById('menu-categories').onclick = () => {
-        document.getElementById('categories').classList.toggle('fly-in');
-    };
+document.getElementById("menu-categories").onclick = () => {
+    document.getElementById("categories").classList.toggle("fly-in");
+};
 
-    document.getElementById('filter-options-menu').onclick = () => {
-        document.getElementById('filter-options').classList.toggle('come-in');
-    };
+document.getElementById("filter-options-menu").onclick = () => {
+    document.getElementById("filter-options").classList.toggle("come-in");
+};
+
+document.getElementById("close-filter-options").onclick = () => {
+    document.getElementById("filter-options").classList.remove("come-in");
+};
 
-    document.getElementById('close-filter-options').onclick = () => {
-        document.getElementById('filter-options').classList.remove('come-in');
+const filterCheckboxes = document.querySelectorAll(
+    "#filter-options input[type=checkbox]"
+);
+
+filterCheckboxes.forEach((filterCheckbox) => {
+    filterCheckbox.onchange = () => {
+        updateFilterNumbers();
     };
+});
 
-    const filterCheckboxes = document.querySelectorAll(
-        '#filter-options input[type=checkbox]'
-    );
-    filterCheckboxes.forEach((filterCheckbox) => {
-        filterCheckbox.onchange = () => {
-            updateFilterNumbers();
-        };
-    });
+function updateFilterNumbers() {
+    const form = document.querySelector("#filter-options form");
+    const formData = new FormData(form);
+    const queryString = new URLSearchParams(formData).toString();
 
-    function updateFilterNumbers() {
-        const form = document.querySelector('#filter-options form');
-        const formData = new FormData(form);
-        const queryString = new URLSearchParams(formData).toString();
-        fetch(
-            `https://cmdmethods.onrender.com/api/filter-numbers?${queryString}`
-        )
-            .then((response) => response.json())
-            .then((filterNumbers) => {
-                const list = document.querySelectorAll('.filterNumber');
-                list.forEach((el) => {
-                    if (filterNumbers[el.dataset.filter] != null) {
-                        if (filterNumbers[el.dataset.filter] === 0) {
-                            el.parentElement.previousElementSibling.disabled = true;
-                        } else {
-                            el.parentElement.previousElementSibling.disabled = false;
-                        }
-                        el.textContent = `(${
-                            filterNumbers[el.dataset.filter]
-                        })`;
+    fetch(`/api/filter-numbers?${queryString}`)
```

Comment:
Dropped the hardcoded `cmdmethods.onrender.com`, as that URL doesn't have a CORS header that allows for localhost usage. Happy to revert if need be.